pkg/skaffold/build/scheduler_test.go

```diff
@@ -360,3 +407,18 @@ func setUpChannels(n int) []chan string {
 	}
 	return outputs
 }
+
+func initializeEvents() {
+	event.InitializeState(latest.Pipeline{
+		Deploy: latest.DeployConfig{},
+		Build: latest.BuildConfig{
+			BuildType: latest.BuildType{
+				LocalBuild: &latest.LocalBuild{},
+			},
+		},
+	},
+		"temp",
+		true,
+		true,
+		true)
```

Comment:
This formatting is a bit weird.  Maybe assign the pipeline to a variable so that the `InitializeState` is one line?